src/django/api/serializers/v1/opensearch_common_validators/date_range_validator.py

```diff
@@ -0,0 +1,21 @@
+from api.serializers.v1.opensearch_validation_interface \
+    import OpenSearchValidationInterface
+
+
+class DateRangeValidator(OpenSearchValidationInterface):
+    def validate_opensearch_params(self, data):
+        errors = []
+
+        date_gte = data.get('date_gte')
+        date_lt = data.get('date_lt')
+
+        if date_gte and date_lt and date_gte > date_lt:
+            errors.append({
+                "field": "date_gte",
+                "message": (
+                    "The 'date_gte' must be "
+                    "less than or equal to 'date_lt'."
+                )
+            })
```

Comment:
We don't need this validation here (and throwing errors on validators level) because we want to collect error messages later by writing them into `errors` list.